examples/stock/company.v.py

```diff
@@ -1,21 +1,23 @@
 # Own shares of a company!
 company: address
-avail_shares: num
-total_shares: num
-price: wei_value
+total_shares: currency_value
+price: num (wei / currency)
 
```

Comment:
The estimates are definitely non-zero for `@constant` functions, I guess I was mistaken to think they'd cost zero gas. In my head that made sense though, the blockchain data is publicly available and should in theory not cost anything to compute something that doesn't modify state. Is this a bug with the gas estimation?